Intel471 threat intel c2 c (#4845)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md

  <p>Collects a list of users from the Google Workspace Users API.</p>
  </div>
</div>
<div className="box smallbox card">
  <div className="container">

>   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="180"/><h4>Intel 471 Threat Intel</h4></a>

Reviewer: ```suggestion
  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel 471 Threat Intel</h4></a>
```

>   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel 471 Threat Intel</h4></a>

Reviewer 2: ```suggestion
  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel471 Threat Intel</h4></a>
```

>   <p>Collects threat indicators from the Intel 471 API.</p>

Reviewer 2: ```suggestion
  <p>Learn to collect threat indicators from the Intel471 platform.</p>
```
